src/interpreter_funcs/voxel_boolean_difference.rs

```diff
@@ -28,6 +31,11 @@ impl fmt::Display for FuncBooleanDifferenceError {
             FuncBooleanDifferenceError::EmptyVoxelCloud => {
                 write!(f, "The resulting voxel cloud is empty")
             }
+            FuncBooleanDifferenceError::TooManyVoxels(max_count, x, y, z) => write!(
+                f,
+                "Too many voxels. Limit set to {}. Try setting voxel size to [{}, {}, {}] or more.",
+                max_count, x, y, z
+            ),
```

Comment:
Note that you are formatting these floating point numbers to have many fractional digits, but the UI is only formatted to take in 3, meaning the user can not actually adhere to the suggestion this error gives.

This would format the output to have 5 fractional digits: `println!("{:.5}", 0.123456);`

Have a look at the `fmt` module for formatting options https://doc.rust-lang.org/stable/std/fmt/index.html